wear/src/main/java/com/example/util/simpletimetracker/presentation/remember/currentActivities.kt

```diff
@@ -35,9 +37,17 @@ import kotlinx.coroutines.async
  * from the phone.
```

Comment:
I should add affectCurrentActivities to the doc if the changes I've made in this file align with your thinking on organization. I'm still getting accustomed with this style of code structuring